src/Wig.Zstd.Tests/DummyFacts.cs

```diff
@@ -0,0 +1,20 @@
+﻿// --------------------------------------------------------------------------------------------------------------------
+// <copyright file="DummyFacts.cs" company="Simply Effective Solutions">
+//   Copyright (c) 2008 - GFG Simply Effective Solutions. All rights reserved.
+// </copyright>
+// --------------------------------------------------------------------------------------------------------------------
+
+
+namespace Wig.Zstd.Tests
```

Comment:
The project should be called wig.Tests (wig is all lowercase)

Reply:
We need to test different options scenario. What happens if the user asks to compress and decompress at the same time?

What happens if they ask to compress a file that does not exist?

What if the output folder does not exist?

What if they ask for a compression level that does not exist?